cni/cni.go

```diff
@@ -355,6 +355,16 @@ func (config *NetworkConfig) GetEndpointInfo(
 		logrus.Debugf("Created raw policy from mapping: %+v --- %+v", mapping, policy)
 		epInfo.Policies = append(epInfo.Policies, policy)
 
+		if config.OptionalFlags.EnableDualStack {
+			v6flags := flags | 3
```

Comment:
Does 3 here represent Ipv6 support? Isn't there a constant defined in hcsshim for this

Reply:
good catch. just updated the PR with the hcsshim flags